en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md

```diff
@@ -1,33 +1,35 @@
 # Choreo Connect Supported Features
 
-Choreo Connect's most recent version (v1.0.0) does not support all of the API Manager's functionalities. In contrast to the default API Gateway, Choreo Connect (the API Microgateway) only offers a fraction of the API Manager product's functionality. The following is a list of Choreo Connect's important features.
+Choreo Connect's most recent version (v1.0.0) does not support all of WSO2 API Manager's (WSO2 API-M) functionalities. In contrast to the default API Gateway, Choreo Connect (the API Microgateway) only offers a fraction of WSO2 API Manager product functionalities. 
 
+## Supported Features
 
-- API Manager as Control Plane
-    - Deploy and manage APIs (REST, Websocket)
-    - Deploy APIs with prototype endpoints
-    - Test APIs with internal test keys from publisher portal
-    - Subscription blocking/unblocking from publisher portal
-    - Applying CORS configuration from publisher portal
-    - Key managers can be added from API Manager admin portal
-    - Websocket APIs (JWT authentication and throttling supported)
-    - Support virtual hosts(Vhosts) to expose the APIs
-    - Support publishing APIs to multiple gateway environments
+The following is a list of Choreo Connect's important features.
 
+- API Manager as a Control Plane
+    - Deploying and managing APIs (REST, WebSocket)
+    - Deploying APIs with prototype endpoints
```

Comment:
@pubudu538 , prototype endpoint was removed in 4.1.0. Is it ok to change this as follows. Because in WSO2 APIM  4.0.0 users can still deploy Prototype APIs with a mock implementation or an actual backend.

"Deploying Prototype APIs"

Reply:
In CC mgw we only support prototype APIs with endpoints.  We don't support mock implementation. So we can do as follows.

4.0.0 docs - Deploying APIs with prototype endpoints
4.1.0 docs - Deploying Prototype APIs

For 4.1.0 we are planning to implement the mock implementation.

Reply:
fixed in https://github.com/wso2/docs-apim/pull/5031/commits/16ad56d574528639751743a43fada3d4f9896c23